validphys2/src/validphys/filters.py

```diff
@@ -239,9 +366,39 @@ def _filter_closure_data(
         total_cut_data_points += ncut
 
     if fakenoise:
-        # ======= Level 1 closure test =======#
 
-        closure_data = make_level1_data(data, closure_data, filterseed, experiments_index, sep_mult)
+        #======= Level 1 closure test =======#
+        closure_data = make_level1_data(
```

Comment:
Process the data and call `make_level1_data` with the data already processed. Passing through this many arguments is asking for trouble down the line (believe me!)